crane-cloud/frontend
# User activity/logs

# Description

Add user activity logs
## Type of change


## Trello Ticket ID
https://trello.com/c/CfWvADHH

## How Can This Been Tested?
Check into this branch and click the user dropdown in the header and select activity on the context menu.

## Checklist:


# Screenshots
![Screenshot from 2022-11-08 17-02-32](https://user-images.githubusercontent.com/32802973/200586402-c8eb0fc4-3c42-4470-9d3d-6a7abb0da278.png)
![Screenshot from 2022-11-08 17-02-18](https://user-images.githubusercontent.com/32802973/200586415-f9a24df1-6b90-405b-86ec-c24f5259398b.png)
